### ISSUE TITLE
*.zip files are 1st synced and then deleted over and over

### ISSUE THREAD

**Issue by ghrasc** [stefankueng/CryptSync#70]
a source folder containing one or more *.7z is synced to the target folder, immediately thereafter the *.7z is/are deleted from the target folder.
Log file looks like
```
09/22/21 08:33:54 : INFO:    Starting CryptSync
09/22/21 08:33:56 : INFO:    syncing folder orig "C:\Temp\Source" with crypt "C:\Temp\Target"
09/22/21 08:33:56 : INFO:    settings: encrypt names: no, use 7z: yes, use GPG: no, use FAT workaround: yes, sync deleted: yes
09/22/21 08:33:56 : INFO:    copy file C:\Temp\Source\SampleFile1.7z to C:\Temp\Target\SampleFile1.7z
09/22/21 08:33:56 : INFO:    copy file C:\Temp\Source\SampleFile2.7z to C:\Temp\Target\SampleFile2.7z
09/22/21 08:33:56 : INFO:    copy file C:\Temp\Source\anotherDir\SampleFile3.7z to C:\Temp\Target\anotherDir\SampleFile3.7z
09/22/21 08:33:56 : INFO:    counterpart of file SampleFile1 does not exist in src folder, delete file
09/22/21 08:33:56 : INFO:    counterpart of file SampleFile2 does not exist in src folder, delete file
09/22/21 08:33:56 : INFO:    counterpart of file anotherDir\SampleFile3 does not exist in src folder, delete file
09/22/21 08:33:56 : INFO:    finished syncing folder orig "C:\Temp\Source" with crypt "C:\Temp\Target"

```
over and over.

Renaming the files to any other file extension (e.g. SampleFile1.myzip) solves the issue.
I guess until this is solved one should use _SyncPair7zExt0 == 0_ (uncheck the box "Use .7z instead of .cryptsync ...")

System:
Win7x64 SP1 ENU

P.S. How to reproduce?
- take a clean machine with Cryptsync
- create 2 empty folders: "Source" and "Target"
- (optional: copy several files into "Source")
- add 1 or more *.7z to source
- configure CryptSync to encrypt and compress "Source" to "Target", "sync unencrypted to encrypted only"
- enable/run the configured sync
- inspect the log file, search for "does not exist in src folder, delete file"

**Comment by stefankueng** [comment]
but you also got `*.7z` in the list of files to copy only and not compress/encrypt?

**Comment by ghrasc** [comment]
Yes, I 1st got "\*.\*" in "_files you don't want encrypted but copied_", now only "\*".
Please find a sample config at https://ufile.io/8vxqd1uf (only 2kB, so "slow download" is good - in case you want to directly import pls see https://github.com/stefankueng/CryptSync/issues/35#issuecomment-925509019 for the HowTo).

Another 'funny' workaround I found today: When enabling _SyncPairEncnames0_ the bug does _not_ appear to happen. Maybe that helps finding the issue.

P.S. Some runs ("_Interval for full scans_" is over) it only copies the files, in the next run it only deletes.
Some other runs show the log from 1st post where copying and deletion happen immediately after another.
